drivers/sensor/ccs811/Kconfig

```diff
@@ -0,0 +1,68 @@
+# Kconfig - CCS811 Digital Gas sensor configuration options
+
+#
+# Copyright (c) 2018 Linaro Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CCS811
+	bool
+	prompt "CCS811 Digital Gas Sensor"
+	depends on I2C
+	default n
+	help
+	  Enable driver for CCS811 Gas sensors.
+
+config CCS811_NAME
+	string
+	prompt "Driver name"
+	default "CCS811"
+	depends on CCS811
+	help
+	  Device name with which the CCS811 sensor is identified.
+
+config CCS811_I2C_ADDR
+	hex
+	prompt "I2C address for CCS811 Sensor"
+	depends on CCS811
+	default "0x5B"
+	help
+	  I2C address of the CCS811 sensor.
+
+	  0x5A: ADDR pin connected to GND.
```

Comment:
Remove empty line L32. Please fix help about address selection, e.g. 0x5a: I2C_ADDR connected to GND, 0x5b: I2C_ADDR connected to VDD

Reply:
Ack